examples/auto-relay/other-node.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
```

Comment:
Instead of `other-node` perhaps we should use the same format as other examples: `dialer`, `listener`?